### ISSUE TITLE
🐛 Bug Report: Getting not found when hovering over users/verified custom permission

### ISSUE THREAD

**Issue by dokgu** [appwrite/console#1448]
### 👟 Reproduction steps

On your collection's settings page, click <kbd>+ Add role</kbd> under Permissions and select `Custom permission` and enter `users/verified` as mentioned [here](https://github.com/appwrite/appwrite/discussions/5036#discussioncomment-4786333).

Once you've added that, hover over the text and it says "Not found."

![permissions](https://github.com/appwrite/console/assets/959300/4a9e44a0-ce85-4ce4-ba79-18f4b0d88da4)

### 👍 Expected behavior

Not really sure what I expect to show in there but "Not found" seems misleading.

### 👎 Actual Behavior

Even if the value is a valid [permission string](https://appwrite.io/docs/rest#permissions), it shows as `Not Found`.

### 🎲 Appwrite version

Version 1.3.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Ms-Kulkarni** [comment]
Is this issue is still open to work on ?

**Comment by h1Anonymoush1** [comment]
Hey i never contributed yet and i want this to be my first issue. It looks like the issue is still ‏‏ available, as you see below a empty bubble appears when hovering over. I will investigate and follow up with my findings.

<img width="742" height="296" alt="Image" src="https://github.com/user-attachments/assets/ef68b2cd-6d38-4040-b12c-9b0e423e06f1" />

**Comment by h1Anonymoush1** [comment]
Please assign this to me

**Comment by BretRen** [comment]
This problem does exist, and I was misled by it before.

**Comment by BretRen** [comment]
This problem has been around for ages and nobody seems to have fixed it. If you want you can try it. @h1Anonymoush1

**Comment by joyalgeorgekj** [comment]
I don't see any hover effects in the console!

**Comment by h1Anonymoush1** [comment]
@joyalgeorgekj you are right , what i will fix is the clicking on it shows a empty bubble i think i will add like just in the bubble it should say verified and also unverified for users/unverified.

Now if someone is planning to add in auth a filter option to filter out users that i can add a link that would naviagte to users with filter verified or something like that.

also or i can just make that users/verified should be plain text like any , users, guest

**Comment by h1Anonymoush1** [comment]
@BretRen @joyalgeorgekj 

Can someone review my pr
